devauth/devauth.go

```diff
@@ -1754,3 +1762,35 @@ func (d *DevAuth) updateCheckInTime(
 		}
 	}
 }
+
+func (d *DevAuth) syncCheckInTime(
+	ctx context.Context,
+	checkInTime *time.Time,
+	deviceId string,
+	tenantId string,
+) error {
+	attributes := []model.DeviceAttribute{
+		{
+			Name:        "check_in_time",
+			Description: nil,
+			Value:       checkInTime,
+			Scope:       "system",
```

Comment:
two small and perhaps annoying comments: `const InventoryScopeSystem = "system"` maybe? and double check: `system` is the scope we want right?

Reply:
done (const); and yes, system is the right scope